src/pages/data-management/index.jsx

```diff
@@ -54,6 +55,13 @@ const DataManagementPage = ({ route }) => {
   }, []);
 
   useEffect(() => {
+    if (activeProject?.experiments.length > 0) {
```

Comment:
Suggestion - put this block below the `if` where the checks are made for activeprojectuuid and etc?

Reply:
This is copied from @aerlaut's PR #416 - we should coordinate to reduce merge conflicts